docs/docsite/rst/reference_appendices/release_and_maintenance.rst

```diff
@@ -1,86 +1,94 @@
+
+Work in Collections is tracked within the individual Collection repositories.
+
+You can refer to the :ref:`porting guides<porting_guides>` for tips on updating your Ansible playbooks to run on newer versions. For Ansible 2.10 and later releases, you can install the Ansible package with ``pip``. See :ref:`intro_installation_guide` for details.  For older releases, You can download the Ansible release from `<https://releases.ansible.com/ansible/>`_.
 
 This table links to the release notes for each major Ansible release. These release notes (changelogs) contain the dates and significant changes in each minor release.
 
 ==================================      =================================================
-Ansible Release                         Status
+Ansible Community Package Release       Status
 ==================================      =================================================
-devel                                   In development (2.11 unreleased, trunk)
-`2.10 Release Notes`_                   In development (2.10 alpha/beta)
-`2.9 Release Notes`_                    Maintained (security **and** general bug fixes)
-`2.8 Release Notes`_                    Maintained (security fixes)
-`2.7 Release Notes`_                    Unmaintained (end of life)
-`2.6 Release Notes`_                    Unmaintained (end of life)
-`2.5 Release Notes`_                    Unmaintained (end of life)
-<2.5                                    Unmaintained (end of life)
+`3.0 Release Notes`_                    Current
+`2.10 Release Notes`_                   Unmaintained (end of life)
 ==================================      =================================================
 
 
-.. Comment: devel used to point here but we're currently revamping our changelog process and have no
-   link to a static changelog for devel _2.6: https://github.com/ansible/ansible/blob/devel/CHANGELOG.md
+.. _3.0 Release Notes:
 .. _2.10 Release Notes:
 .. _2.10: https://github.com/ansible-community/ansible-build-data/blob/main/2.10/CHANGELOG-v2.10.rst
-.. _2.9 Release Notes:
-.. _2.9: https://github.com/ansible/ansible/blob/stable-2.9/changelogs/CHANGELOG-v2.9.rst
-.. _2.8 Release Notes:
-.. _2.8: https://github.com/ansible/ansible/blob/stable-2.8/changelogs/CHANGELOG-v2.8.rst
-.. _2.7 Release Notes: https://github.com/ansible/ansible/blob/stable-2.7/changelogs/CHANGELOG-v2.7.rst
-.. _2.6 Release Notes:
-.. _2.6: https://github.com/ansible/ansible/blob/stable-2.6/changelogs/CHANGELOG-v2.6.rst
-.. _2.5 Release Notes: https://github.com/ansible/ansible/blob/stable-2.5/changelogs/CHANGELOG-v2.5.rst
 
 
-ansible-core release cycle
---------------------------
+Release cycle for ansible-base/ansible-core
+-------------------------------------------
```

Comment:
Do you think this might be clearer:
```suggestion
Release cycle for ansible-base & ansible-core
---------------------------------------------
```

Reply:
maybe to remove `ansible-base` at all? There are only the `ansible-core` notion used in the following sentences

Reply:
Addressed in cca19eef3206585b6f806ace09cba21a993cab28.